app/assets/javascripts/utilities/buildArticleHTML.js.erb

```diff
@@ -113,7 +113,45 @@ function buildArticleHTML(article) {
       timeAgoInWords = timeAgo({ oldTimeInSeconds: article.published_at_int });
     }
 
-    return '<div class="single-article single-article-small-pic" data-content-user-id="'+article.user_id+'">\
+    return '<div class="crayons-story" data-content-user-id="'+article.user_id+'">\
+      '+videoHTML+'\
+      <div class="crayons-story__body">\
+        <div class="crayons-story__top">\
+          <!-- Author -->\
```

Comment:
This is a placeholder for author section.. See /page/crayons or Author.jsx for reference.